### PR TITLE
handle -en adj/adv

### LINKED ISSUE #476: handling of "en" words
see <https://www.notion.so/witty-works/Are-there-exceptions-in-case-of-en-ending-to-be-taking-from-terms-to-alternatives-c68f2475785f47dba5984cd55d305c75>

### PR DESCRIPTION
**Pull request by lsmith77**
fixes https://github.com/witty-works/nlp_api/issues/476

just poking around here .. likely the transformation would have to be added to other functions too

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1089,6 +1089,35 @@ def ing_ify_alternatives(token, alternatives):
...
+"""Function to change adjectives to -en form in alternatives"""
+
+
+def en_ify_alternative(lang, alternative):
+    tokens = model[lang.lang](alternative.rstrip().replace("\n", " "))
+
+    alternative = ""
+    for token in tokens:
+        alternative += token.text
+        if token.text[-1] != "m" and (
```
Comment (lsmith77) [inline app/main.py]: this prevents `mit gutem Ergebnis` to be converted .. not sure if this isn't more an issue with this alternative just not fitting like the others

**Review by lnazarenko — COMMENTED** [review]
If the "en" adding to alternatives should be for the verbs as well? 
Here:  if token.pos_ == "ADJ" or token.pos_ == "ADV" or token.pos_ == "VERB":

**Comment by lsmith77** [comment]
yeah .. added ..

what then remains is deciding in which of the functions we need to apply this on alternatives?

**Review by lnazarenko — APPROVED** [review]
It seems now is good:
en_ify_alternative function adds "en" to single-click alternatives if there are adjectives, adverbs and verbs. 
en_ify_alternatives function changes alternatives by adding "en" if user's words ends with "en" and these words are adjectives, verbs and adverbs

**Comment by lnazarenko** [comment]
This type of alternative transformation needs to be added to ub_words_phrase_matcher_de and it will cover all adjectives, verbs and adverbs from google sheet

**Comment by lsmith77** [comment]
@lnazarenko I had to do some more adjustments but now it seems to be correct. can you check one more time?

**Comment by lsmith77** [comment]
note I don't see anything we can do to make it pass the quality gate. in fact I think its a bug in sonar, since this PR doesn't introduce the duplicate blocks sonar complains about. it just adds to those blocks.

**Review by lnazarenko — COMMENTED** [review]
Why for the word "Kunden" there is empty list of alternatives? I mean this test: Wir suchen Ninja Programmierer für unseere Kunden
It should be some single click alternatives for this word

**Comment by lsmith77** [comment]
FTR I spend a bit of time to reduce code duplication in general, which is always a good idea anyway :)

the list is empty for `Ninja`

https://github.com/witty-works/nlp_api/blob/f62fc305a5b7a1fc343181c112fe848ef4b61e1c/tests/test_1_0/test_api/output.json

**Comment by lnazarenko** [comment]
Ah, yes, right. It was three inspiration alternatives for Ninja before and now the API doesn't show them

**Review by lnazarenko — APPROVED** [review]

**Pull request merged**